### ISSUE TITLE
Your favourite feature requests?

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#2874]
<img  width="150" height="150" src="https://user-images.githubusercontent.com/5811789/135705995-ddd550f6-efce-47ef-a3d8-d1ec7ae648fe.png">

## Let me know your favourite feature requests

Please find your favourite feature request ticket [from feature request list](https://github.com/AndBible/and-bible/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature%22) or from [wishing well project board](https://github.com/orgs/AndBible/projects/6) and copy link / ticket id here. You may also refer to other tickets than above mentioned feature request tickets, and feel free to create new feature request tickets too if you can't find what you'd like to have.

If/when I [receive enough financial support](https://shop.andbible.org), I will  make a poll in our [Telegram/Matrix group](https://github.com/AndBible/and-bible/wiki/Support) of selected items and  based on that will make my near-term roadmap (after 5.0 release) that is implemented in paid (non-earmarked) hours.

**Comment by fco3jo-mexico** [comment]
Good day, everyone.
I would have to know how long we can have modules in Spanish, it is a good tool but too short in terms of usefulness since not having modules in Spanish is only reduced to a simple reader of Bible and nothing more

*Dev note: Module development is out of our scope. Please see [FAQ](https://github.com/AndBible/and-bible/wiki/FAQ#please-add-module-x-to-andbible) on how to proceed with getting more modules to AndBible.*

**Comment by timunrau** [comment]
- https://github.com/AndBible/and-bible/issues/2615
- https://github.com/AndBible/and-bible/issues/2547

**Comment by tuomas2** [comment]
- #790

**Comment by pepaulo** [comment]
- #2886

**Comment by Noitarud** [comment]
If I can only pick one (and that is not mine), [#186](https://github.com/AndBible/and-bible/issues/186#issuecomment-1231508059) - search book group. Added my own input.
- #186

**Comment by meichthys** [comment]
<s>* [Nextcloud Sync #2615](https://github.com/AndBible/and-bible/issues/2615)</s>
Now that my favorite feature is implemented, I'm picking a new one :) #3586 (Memorization Tracking)

**Comment by jeebsdotorg** [comment]
https://github.com/AndBible/and-bible/issues/2142

- That one is an OCD thing.  I like programs to manually start their execution afresh, and not give so much trying to being "smart".
- You should see my erratic history on all my Workspaces, by just simple straightforward reads through.
- If I want something in my history, I will bookmark it for the next execution of the program, and use that feature instead.
- Fun fact:  This issue also affected Xiphos the last time I studied on the computer (2017).  Maybe it still does.  It's been around for a while.  I would be in favor of the complete removal of the history feature, in-fact, but I can see it handy for some folks in some cases.


https://github.com/AndBible/and-bible/issues/2896

- This doesn't even need apply to just the "Speak" features, which I am playing with now.  Widgets are a powerful and underutilized example of how Android could be a better operating system.
- Now, that I pickup on following a reading plan, a synthesized voice may help me get through the passages more quickly, with better comprehension by reading along.  But, to read along with the widget is near impossible for the next bulletpoint.
- I use GrapheneOS, which ships with a totally black background.  Boring.  So I found an image of a creed (text, like the Apostle's Creed), and made that my background...  it is still mostly a black background, but now with a lot of white text on it that I arrange any icons and widgets around for cosmetic purposes...
- The reason I can not read along with it now, via the widget, is because there seemingly are no options or preferences to control the opacity and transparency of the widget's background.  I would love to see a background with the scripture being read in the widget area, with a foreground color that could also be controlled, along with transparency/opacity options for those who may think lacking transparency equates to being "old".  Sometimes that sort of thing drives people away from otherwise great projects.  Less modern looks-and-feels are better, in my opinion...
- What the widget yields me is a bunch of unreadable overlapping text with my wallpaper's text containing the creed, in slightly different fonts, sizes, and color.  It's close enough that it looks like a jumble of overlapping words.
- If I could turn my third Android workspace's background to just plain white, that would also solve the problem.  I don't believe Android has this capability.  Even getting wallpaper aspect ratios that do not match the device's ratio cause a great mess, with Android trying to be "smart", by scrolling the wallpaper image just a little bit, by a little bit more, backward and forward, as I navigate across workspaces (not in the And Bible sense) to use my device like a human.
- In a nutshell, Android is a bit cuckoo ;).  Even Google's programs lack, way more often than not, what could be helpful and useful widgets.  I have no awareness of what you go through to create widgets for programs on Android...  but, mad props to the And Bible team that you all even have widgets at all.  Seems like a lot of the Android ecosystem could be drastically enhanced with more widgets and more configurable widgets!

**Comment by tuomas2** [comment]
Thanks @jeebsdotorg. I copied the comments to the relevant tickets so they are not forgotten here.

**Comment by Dometalican** [comment]
Honestly, for me, I would love to see this app gain a widget that can be resized for just getting a 'verse of the day' that rotates automatically at midnight local time.

I want to not use the Youversion app for this as it has not been working as well lately and it is FULL of trackers. Definitely prefer this app.

**Comment by meichthys** [comment]
> Honestly, for me, I would love to see this app gain a widget that can be resized for just getting a 'verse of the day' that rotates automatically at midnight local time.
> 
> I want to not use the Youversion app for this as it has not been working as well lately and it is FULL of trackers. Definitely prefer this app.

You can follow any progress on this feature here: https://github.com/AndBible/and-bible/issues/184

Also, if an estimate is given by the developer, you can sponsor the feature by supporting the developer for the time required to implement it.

**Comment by b-risk** [comment]
Would love to see Material 3, also with Material You Expressive being out now #2624
